_glossary/service-producer.md

```diff
@@ -2,4 +2,4 @@
 title: Service Producer
 type: markdown
 ---
-The agent that creates a service by writing source code.
+The agent that creates a **Service** and manages which **Service Consumers** may use it.
```

Comment:
We're talking about different roles here, it is the same person, but in their provider role that they are deciding who can access, not as operator.  Or at least that's what we usually mean by the producer/consumer/operator split.  We could maybe just delete producer/consumer from the glossary since they're more about API management than service management?

Reply:
Yeah, I think removing these for the time being would be good. We'll have a better grip on things for 0.3